### DISCUSSION TITLE
🐛 Bug: Mixed Content Error

### DISCUSSION THREAD

**Discussion by Asanatsa** [nocodb/nocodb#11864] [General]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Using Apache reverse proxy nocodb (force https)

### Desired Behavior

load picture correctly
etc.

### Project Details

version 0.203.2
docker image

### Attachments

![屏幕截图 2024-01-08 002503](https://github.com/nocodb/nocodb/assets/70144744/91c95bb4-2f60-469d-a0b4-d6294cbfa661)

**Comment by nktnet1** [comment]
Ran into a similar issue with exporting CSV. Reason was because a reverse proxy (Traefik) was being used. I also had a Cloudflare Tunnel setup with the same issue.

The fix was simply to set the `NC_PUBLIC_URL` environment variable, e.g.

```env
NC_PUBLIC_URL=https://nocodb.yourdomain.com
```

For more information, see here:
- https://nocodb.com/docs/self-hosting/environment-variables#frontend

    <img width="800px" alt="nc-public-url-environment-variable-docs" src="https://github.com/user-attachments/assets/31a70fc3-c5c2-4970-ae41-ce5a22c738b4" />
